SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -83,6 +83,34 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 			VerifyAndStartDeployment();
 		});
 	}
+
+#if PLATFORM_WINDOWS
```

Comment:
I would prefer to keep this code in LocalDeploymentManager files. As an option, this code may be moved to a new Init() or Start() function that takes ExposedRuntimeIP as an argument and is called here.

Reply:
Agreed. I've updated it now.